apisec/internal/timed/clock.go

```diff
@@ -0,0 +1,42 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2023-present Datadog, Inc.
+
+package timed
+
+import "time"
+
+// biasedClock is a specialized clock implementation used to ensure we can get
+// 32-bit wide timestamps without having to worry about wraparound.
+type (
+	ClockFunc = func() int64
+
+	biasedClock struct {
```

Comment:
```suggestion
type (
	ClockFunc = func() int64

	// biasedClock is a specialized clock implementation used to ensure we can get
	// 32-bit wide timestamps without having to worry about wraparound.
	biasedClock struct {
```
Right now the doc shows in the wrong place in the UI